app/views/advanced_messenger/_html_head.html.erb

```diff
@@ -1,15 +1,243 @@
+<%# ============= Plugin styles  ============= %>
 <% content_for :header_tags do %>
     <%= stylesheet_link_tag 'advanced_messenger', :plugin => 'redmine_advanced_messenger' %>
 <% end %>  
+<%# ============= Global unread notifications indicator located on main title bar  ============= %>
 <%= javascript_tag do%>
```

Comment:
I think you should add the

```ruby
if (AdvancedMessengerSettings.get_setting(:show_unread_notifications) == '1'
```

check to this block as well. What do you think?

Reply:
Indeed. I changed and included this block inside the check